web/styles/widgets.css

```diff
@@ -270,10 +270,10 @@ input {
 
 .poll-edit-question,
 .todo-edit-task-list-title {
-    opacity: 0.4;
+    color: var(--color-message-action-visible);
 
     &:hover {
```

Comment:
Should this have `:focus-visible` too?

Reply:
Can you please explain this a bit more?

Reply:
I think I would need a suggestion regarding this.

Reply:
I think yes, it should have `:focus-visible` too. It would improve accessibility for the users who navigate through keyboard or some devices.